pkg/events/core.go

```diff
@@ -10528,6 +10538,120 @@ var CoreEvents = map[ID]Definition{
 		},
 	},
 	//
+	// Begin of Signal Events (Control Plane)
+	//
+	SignalCgroupMkdir: {
+		id:      SignalCgroupMkdir,
+		id32Bit: Sys32Undefined,
+		name:    "signal_cgroup_mkdir",
+		dependencies: Dependencies{
+			probes: []Probe{
+				{handle: probes.SignalCgroupMkdir, required: true},
+			},
+		},
+		sets: []string{},
+		params: []trace.ArgMeta{
+			{Type: "u64", Name: "cgroup_id"},
+			{Type: "const char*", Name: "cgroup_path"},
+			{Type: "u32", Name: "hierarchy_id"},
+		},
+	},
+	SignalCgroupRmdir: {
+		id:      SignalCgroupRmdir,
+		id32Bit: Sys32Undefined,
+		name:    "signal_cgroup_rmdir",
+		dependencies: Dependencies{
+			probes: []Probe{
+				{handle: probes.SignalCgroupRmdir, required: true},
+			},
+		},
+		sets: []string{},
+		params: []trace.ArgMeta{
+			{Type: "u64", Name: "cgroup_id"},
+			{Type: "const char*", Name: "cgroup_path"},
+			{Type: "u32", Name: "hierarchy_id"},
+		},
+	},
+	SignalSchedProcessFork: {
+		id:      SignalSchedProcessFork,
+		id32Bit: Sys32Undefined,
+		name:    "signal_sched_process_fork",
+		dependencies: Dependencies{
+			probes: []Probe{
+				{handle: probes.SignalSchedProcessFork, required: true},
+			},
+		},
+		sets: []string{},
+		params: []trace.ArgMeta{
+			{Type: "int", Name: "parent_tid"},
+			{Type: "int", Name: "parent_ns_tid"},
+			{Type: "int", Name: "parent_pid"},
+			{Type: "int", Name: "parent_ns_pid"},
+			{Type: "unsigned long", Name: "parent_start_time"},
+			{Type: "int", Name: "child_tid"},
+			{Type: "int", Name: "child_ns_tid"},
+			{Type: "int", Name: "child_pid"},
+			{Type: "int", Name: "child_ns_pid"},
+			{Type: "unsigned long", Name: "child_start_time"},
```

Comment:
Since its a new event, we can control different arguments for the signal ebpf programs (I think this is what was requested in the other review as well).